gpt4all-chat/metadata/models2.json

```diff
@@ -237,5 +237,21 @@
     "url": "https://huggingface.co/TheBloke/em_german_mistral_v01-GGUF/resolve/main/em_german_mistral_v01.Q4_0.gguf",
     "promptTemplate": "USER: %1 ASSISTANT: ",
     "systemPrompt": "Du bist ein hilfreicher Assistent. "
+  },
+  {
+    "order": "g",
+    "md5sum": "02bc0d06f6b453ff0db6870a624a22eb",
+    "name": "Ghost 7B v0.9.1",
+    "filename": "model.Q4_K_M.gguf",
+    "filesize": "4368439488",
+    "requires": "2.5.0",
+    "ramrequired": "8",
+    "parameters": "7 billion",
+    "quant": "q4_k_m",
+    "type": "Mistral",
+    "description": "<strong>Ghost 7B v0.9.1</strong> fast, powerful and smooth for Vietnamese and English languages.",
+    "url": "https://huggingface.co/lamhieu/ghost-7b-v0.9.1-gguf/resolve/main/model.Q4_K_M.gguf",
```

Comment:
Why not use https://huggingface.co/lamhieu/ghost-7b-v0.9.1/resolve/main/model-q4_0.gguf as it will be accelerated as @cebtenzzre says?

Reply:
hi @manyoso , It was a mistake in the name when I built it, it was `Q4_K_M` not `Q4_0`. I'll find a way to make a new one as soon as possible to be better (but let's get everything working with gguf on gpt4all first).

Reply:
We can approve this when you have the Q4_0 ready but all the models that we have available are Q4_0 at the moment so I think we should wait until you have one ready.

Reply:
@manyoso I resolved this comment, using Q4_0 version. Thanks for review!